### DISCUSSION TITLE
Clean cache

### DISCUSSION THREAD

**Discussion by edersilvago** [amanharwara/altus#331] [General]
With continued use of Altus, it stores temporary files cached on the computer. How do I remove it? I used a tool to clear browser cache files, but I didn't get satisfactory results.

**Comment by amanharwara** [comment]
Depending on the platform you can delete folders with `Cache` in their name from the Altus config folder:

-  `%APPDATA%\Altus` on Windows
-  `$XDG_CONFIG_HOME/Altus` or `~/.config/Altus` on Linux
-  `~/Library/Application Support/Altus` on macOS

**Comment by edersilvago** [comment]
I compared 2 computers in the same folder, they have the same file sizes, I imagine the cache is in another location. Using Windows 10.
![Screenshot_1](https://github.com/user-attachments/assets/12567e2d-1e0c-47c8-a55f-fb8733f24b93)

**Comment by edersilvago** [comment]
I found where the folder with the Altus cache is, it creates a folder with a random name and saves the files there.
![image](https://github.com/user-attachments/assets/a21e559a-7129-4f63-bac4-809aedacd492)
